pennylane/ops/qubit/parametric_ops_multi_qubit.py

```diff
@@ -754,40 +867,262 @@ def compute_decomposition(
 
         Returns:
             list[Operator]: decomposition of the operator
-        """
-        phi = params[0]
-        k, n = hyperparams["dimension"]
-
-        def _get_op_from_binary_rep(binary_rep, theta, wires):
-            if len(binary_rep) == 1:
-                op = (
-                    PhaseShift(theta, wires[0])
-                    if int(binary_rep)
-                    else PauliX(wires[0]) @ PhaseShift(theta, wires[0]) @ PauliX(wires[0])
-                )
-            else:
-                base_op = (
-                    PhaseShift(theta, wires[-1])
-                    if int(binary_rep[-1])
-                    else PauliX(wires[-1]) @ PhaseShift(theta, wires[-1]) @ PauliX(wires[-1])
-                )
-                op = qml.ctrl(
-                    base_op, control=wires[:-1], control_values=[int(i) for i in binary_rep[:-1]]
-                )
-            return op
 
-        n_log2 = int(np.log2(n))
-        positive_binary_reps = [bin(_k)[2:].zfill(n_log2) for _k in range(k)]
-        negative_binary_reps = [bin(_k)[2:].zfill(n_log2) for _k in range(k, n)]
+        For this decomposition, we will decompose the generator of the PCPhase gate,
+        which we denote as :math:`G`, into multiple projectors, which generate (multi-controlled)
+        phase shift gates. The generator of ``PCPhase`` is given as (see :meth:`~.generator`)
+
+        .. math::
+
+            G = 2 \left(\sum_{j=0}^{d-1} |j\rangle\langle j|\right) - \mathbb{I}_N
+
+        where we denoted the overall dimension as :math:`N=2^n` for :math:`n` qubits and the
```

Comment:
mention that the $j$ in $|j\rangle$ is binary

Reply:
might also help to actually write the matrix

Reply:
Do we need the $j$ in an explicit binary bit string format? :thinking: 
I added a description of the matrix via $\text{diag}(\underset{d \text{ times}}{\underbrace{1,\dots,1}},\underset{(N-d) \text{ times}}{\underbrace{-1,\dots,-1}})$ :)